components/brave_rewards/resources/ui/components/rewardsButton/index.tsx

```diff
@@ -27,7 +27,7 @@ export default class RewardsButton extends React.PureComponent<Props, {}> {
 
     return (
       <StyledButtonWrapper
-        type={type}
```

Comment:
note for reviewer: i had to change the prop name from `type` to `buttonType` because `type` is a reserved property of HTML buttons. this caused typescript to interpret `type` as undefined regardless of the actual prop type definition. (see https://stackoverflow.com/questions/62679634/type-string-is-not-assignable-to-type-undefined)